src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -105,6 +113,7 @@ public FilterPanel addPanelAt(int index) {
 
         updatePanelIndices();
         setCurrentlySelectedPanel(Optional.of(index));
+        mainStage.show();
```

Comment:
This is to fix the ```NoNodesVisibleException``` caused when adding a new panel. I guess there is a way around ```PanelControl``` calling the stage, by creating a method in ```UI``` for showing mainstage.